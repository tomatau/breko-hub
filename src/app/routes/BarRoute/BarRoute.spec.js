import React from 'react'
import ConnectedComponent from './BarRoute'

const { WrappedComponent: BarRoute } = ConnectedComponent

describe(`BarRoute Component`, function () {
  helpers.setupSnapshots(__filename)

  it(`matches snapshot`, () => {
    expect(
      snap(shallow(<BarRoute />))
    ).to.matchSnapshot()
  })
})
