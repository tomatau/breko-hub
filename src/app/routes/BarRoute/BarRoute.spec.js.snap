// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BarRoute Component matches snapshot 1`] = `
<Fragment>
  <Helmet
    defer={false}
    encodeSpecialCharacters={true}
  >
    <title>
      Bar
    </title>
  </Helmet>
  <section
    className="BarRoute"
  >
    <h2>
      The Bar Route!
    </h2>
    <p>
      API request to the 
      <code>
        /api/bar
      </code>
       endpoint.
    </p>
    <div
      aria-live="polite"
      className="src-app-routes-BarRoute-BarRoute-module-block"
    />
  </section>
</Fragment>
`;
